tests/IncludeRelationsCountTest.php

```diff
@@ -52,7 +51,7 @@ public function it_throws_an_exception_if_attempting_to_include_relations_count_
 
         $this->app->instance(DynamicRelationsIncludeRequest::class, $mock);
 
-        $this->expectException(LoadablesAreNotDefinedException::class);
+        // $this->expectException(LoadablesAreNotDefinedException::class);
```

Comment:
these tests can be deleted since they are not testing anything now.

Reply:
Yes.